samples/pmic/npm6001/prj.conf

```diff
@@ -0,0 +1,27 @@
+#
+# Copyright (c) 2022 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+# Enable I2C ("TWI")
+CONFIG_I2C=y
+
+# Enable nPM6001 library
+CONFIG_LIB_NPM6001=y
+# Needed for certain nPM6001-EK revisions
+CONFIG_LIB_NPM6001_C3_WORKAROUND=y
+
+# Enable logging
+CONFIG_LOG=y
+CONFIG_LOG_BACKEND_RTT=n
+
+# Enable Shell
+CONFIG_SHELL=y
+
+# Ease debugging
+CONFIG_DEBUG_OPTIMIZATIONS=y
```

Comment:
```suggestion
```

this should not be defaut, you can provide an extra `debug.conf` with debug settings if needed